scripts/solve_network.py

```diff
@@ -1048,9 +1050,9 @@ def solve_network(n, config, solving, **kwargs):
         snakemake = mock_snakemake(
             "solve_network",
             simpl="",
-            clusters="54",
+            clusters="10",
             ll="copt",
-            opts="Co2L-1H",
+            opts="Co2L-3H",

```

Comment:
Is this change needed? Absolutely agree though that we must improve using wildcards across the scripts, and there is a new issue #1197 to track that

Reply:
I will revert this to before. I used this while testing locally.